tests/sentry/api/endpoints/test_system_options.py

```diff
@@ -69,7 +69,7 @@ def test_put_unknown_option(self):
 
     def test_put_simple(self):
         self.login_as(user=self.user, superuser=True)
-        assert options.get("mail.host") != "lolcalhost"
-        response = self.client.put(self.url, {"mail.host": "lolcalhost"})
+        assert options.get("mail.host") != "localhost"
```

Comment:
I think these were intentional. This test is now failing.

Reply:
Thanks @markstory